samples/api-client/mqtt_example/README.rst

```diff
@@ -0,0 +1,151 @@
+
+#. Install the dependencies needed to run the samples.
+
+    .. code-block:: bash
+
+        $ pip install -r requirements.txt
+
+.. _pip: https://pip.pypa.io/
+.. _virtualenv: https://virtualenv.pypa.io/
+
+Samples
+-------------------------------------------------------------------------------
+
+MQTT example
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+.. image:: https://gstatic.com/cloudssh/images/open-btn.png
+   :target: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=iot/api-client/mqtt_example/cloudiot_mqtt_example.py,iot/api-client/mqtt_example/README.rst
+
+
+
+
+To run this sample:
+
+.. code-block:: bash
+
+    $ python cloudiot_mqtt_example.py
+
+    usage: cloudiot_mqtt_example.py [-h] --algorithm {RS256,ES256}
+                                    [--ca_certs CA_CERTS]
+                                    [--cloud_region CLOUD_REGION] [--data DATA]
+                                    --device_id DEVICE_ID
+                                    [--gateway_id GATEWAY_ID]
+                                    [--jwt_expires_minutes JWT_EXPIRES_MINUTES]
+                                    [--listen_dur LISTEN_DUR]
+                                    [--message_type {event,state}]
+                                    [--mqtt_bridge_hostname MQTT_BRIDGE_HOSTNAME]
+                                    [--mqtt_bridge_port {8883,443}]
+                                    [--num_messages NUM_MESSAGES]
+                                    --private_key_file PRIVATE_KEY_FILE
+                                    [--project_id PROJECT_ID] --registry_id
+                                    REGISTRY_ID
+                                    [--service_account_json SERVICE_ACCOUNT_JSON]
+                                    {device_demo,gateway_send,gateway_listen} ...
+
+    Example Google Cloud IoT Core MQTT device connection code.
+
+    positional arguments:
+      {device_demo,gateway_send,gateway_listen}
+        device_demo         Connects a device, sends data, and receives data.
+        gateway_send        Sends data from a gateway on behalf of a device that
+                            is bound to it.
+        gateway_listen      Listens for messages sent to the gateway and bound
+                            devices.
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      --algorithm {RS256,ES256}
+                            Which encryption algorithm to use to generate the JWT.
+      --ca_certs CA_CERTS   CA root from https://pki.google.com/roots.pem
+      --cloud_region CLOUD_REGION
+                            GCP cloud region
+      --data DATA           The telemetry data sent on behalf of a device
+      --device_id DEVICE_ID
+                            Cloud IoT Core device id
+      --gateway_id GATEWAY_ID
+                            Gateway identifier.
```

Comment:
Seeing this block on the [README preview](https://github.com/googleapis/python-iot/tree/mqtt-example-nox/samples/api-client/mqtt_example), it's hard to parse through the information with so much text into this block. I'm not sure what's the best way to present this in a readable way, perhaps adding a line break in between could help, or maybe in chunks of 4 and then a line break.

Reply:
This whole thing comes from the readme generation, I cannot change it.